### DISCUSSION TITLE
Pair Extraordinaire badge

### DISCUSSION THREAD

**Discussion by sergeevsimon606** [Schweinepriester/github-profile-achievements#418] [Q&A] [answered]
Why can't I get a Pair Extraordinaire badge?
I added a friend during a repository commit, but the score never came :(

Help me!

**Comment by ghost** [comment, marked as answer]
Be sure you added a friend when you commit and _merge with your master branch_. 
If you commit immediately to the main branch without creating a dev branch, for example, then you will not be given a badge.

The flow is: 
_Create a dev branch -> modify the file -> commit using [Coauthored](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) with your friend -> make a merge request with the main branch -> Done! 🎉_

**Reply by sergeevsimon606** [reply]
Thank you! 
I did not make a merge request but committed immediately in the main branch

**Reply by ghost** [reply]
Glad it helped!
